ads/google/a4a/utils.js

```diff
@@ -594,3 +594,94 @@ export function getBinaryTypeNumericalCode(type) {
     'canary': '2',
   }[type] || null;
 }
+
+/** @const {!RegExp} */
+const IDENTITY_DOMAIN_REGEXP_ = /\.google\.(?:com?\.)?[a-z]{2,3}$/;
+
+/** @typedef {{
```

Comment:
Maybe it's my ignorance, but a little more documentation on what an IdentityToken actually is might be helpful here or on one of the two functions below.